cmd/cmd.go

```diff
@@ -445,6 +445,14 @@ func New() *Cmd {
 			} else {
 				options = append(options, server.UseStorage(storage))
 			}
+		case "s3-ec2role": // using AWS IAM Role and EC2 Profile
```

Comment:
this is not really needed.

you should rather add something like `s3-credentials-type` config param, default to `access-key` and refactor https://github.com/dutchcoders/transfer.sh/blob/e97fdcb2930a3d2d139c80ec1cd3caf521d50b7b/server/storage.go#L176 and https://github.com/dutchcoders/transfer.sh/blob/main/server/utils.go#L43